Source/WPEFramework/PluginHost.cpp

```diff
@@ -489,12 +489,23 @@ POP_WARNING()
                 if (_config->Process().Priority() != 0) {
                     myself.Priority(_config->Process().Priority());
                 }
-                if (_config->Process().User().empty() == false) {
-                    myself.User(_config->Process().User());
-                }
+
+                Core::Directory dataPath(_config->DataPath().c_str());
```

Comment:
DataPath is pointing to ROM (Read Only Memory) storage of the setup, so by definition, you should not (and if configured correctly) can not change proeprties on this path. This is something that needs to be done properly at image build time.
So this, if it works, it means that the config is incorrect (DATA Path is not pointing to the ROM storage) and if it solves an issue, it is hiding a root cause that the plugin did not properly use the dataPath. So please do not make any modifications to the DataPath and resolve the root cause in the plugin.

Reply:
moved from here to build system